nc-games/src/components/SingleReview.jsx

```diff
@@ -2,36 +2,57 @@ import { useEffect } from "react";
 import { useParams } from "react-router-dom";
 import { getReviewById } from "../utils/api";
 import { useState } from "react";
+import Comments from "./Comments";
 
-const SingleReview = ({ isLoading, setIsLoading }) => {
+const SingleReview = ({
+  isLoading,
+  setIsLoading,
+  currentVotes,
+  setCurrentVotes,
+}) => {
   const { review_id } = useParams();
   const [singleReview, setSingleReview] = useState({});
+  const [showComments, setShowComments] = useState(false);
+  const [isLoadingReview, setIsLoadingReview] = useState(true);
 
   useEffect(() => {
-    setIsLoading(true);
+    setIsLoadingReview(true);
     getReviewById(review_id).then((review) => {
-      setIsLoading(false);
+      setIsLoadingReview(false);
       setSingleReview(review);
+      setCurrentVotes(review.votes);
     });
-  }, []);
+  }, [review_id, setIsLoadingReview, setCurrentVotes]);
 
-  if (isLoading) {
+  if (isLoadingReview) {
     return <h2>Loading...</h2>;
   } else {
     return (
       <section className="singleReview">
+        {console.log(showComments)}
         <img src={singleReview.review_img_url} alt={singleReview.title} />
         <h2>{singleReview.title}</h2>
         <h3>{singleReview.category} game</h3>
         <h3> Designed by {singleReview.designer}</h3>
         <p>{singleReview.review_body}</p>
         <p> Owner: {singleReview.owner}</p>
-        <p>{singleReview.comment_count} comments</p>
-        {singleReview.votes === 1 ? (
-          <p>{singleReview.votes} vote</p>
+        {currentVotes === 1 ? (
+          <p>{currentVotes} vote</p>
         ) : (
-          <p>{singleReview.votes} votes</p>
+          <p>{currentVotes} votes</p>
         )}
+        <button
+          onClick={() => {
+            setShowComments(true);
+          }}
+        >
+          {singleReview.comment_count} comments
+        </button>
+        <Comments
```

Comment:
Really like that you've broken comments out to a seperate component and used a toggle button to show and hide the comments this makes for a really nice UI